srcpkgs/vala-panel/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'vala-panel'
+pkgname=vala-panel
+version=0.4.87
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="vala-devel cmake-vala gtk+3-devel glib-devel"
```

Comment:
libglib-devel

Reply:
> This was done very poorly with no understanding for core xbps-src concepts like shlib tracking.
> 
> Please apply proposed changes.

sorry I am pretty new to void linux, but I will try to fix the templates